packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -1758,6 +1809,7 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/val
 +  component: kube-controller-manager
 +  clients:
 +    port: 10011
++    useLocalhost: true
```

Comment:
Hardened clusters don't expose the port for `kubeScheduler` and `kubeControllerManager` so the defaults needed to be modified for PushProx.